.github/workflows/near.yml

```diff
@@ -28,12 +28,14 @@ jobs:
           node-version: 12
 
       - name: Install and Build 🔧
+        working-directory: ./near
         run: |
           npm install
           npm run build
 
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@3.7.1
+        working-directory: ./near
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
what about building it like, `/pages/eth` and `/pages/near` as part of the same workflow and then the gh-pages would have both live under two subdirectories?